cmd/juju/user/login_test.go

```diff
@@ -82,30 +82,12 @@ func (s *LoginCommandSuite) TestInit(c *gc.C) {
 	}
 }
 
-func (s *LoginCommandSuite) assertStorePassword(c *gc.C, user, pass string) {
-	details, err := s.store.AccountByName("testing", user)
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(details.Password, gc.Equals, pass)
-}
-
-func (s *LoginCommandSuite) assertStoreMacaroon(c *gc.C, user string, mac *macaroon.Macaroon) {
-	details, err := s.store.AccountByName("testing", user)
-	c.Assert(err, jc.ErrorIsNil)
-	if mac == nil {
-		c.Assert(details.Macaroon, gc.Equals, "")
-		return
-	}
-	macaroonJSON, err := mac.MarshalJSON()
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(details.Macaroon, gc.Equals, string(macaroonJSON))
-}
-
 func (s *LoginCommandSuite) TestLogin(c *gc.C) {
-	context, args, err := s.run(c)
+	context, args, err := s.run(c, "current-user\nsekrit\nsekrit\n")
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(coretesting.Stdout(context), gc.Equals, "")
 	c.Assert(coretesting.Stderr(context), gc.Equals, `
-password: 
+username: password: 
```

Comment:
When we read the username, output is echoed to the terminal; so when the user hits enter, a newline is printed. There's no echo for the password, so we print one out.